packages/craco/lib/features/test/merge-jest-config.js

```diff
@@ -9,8 +8,13 @@ const { projectRoot } = require("../../paths");
 const BABEL_TRANSFORM_ENTRY_KEY_BEFORE_2_1_0 = "^.+\\.(js|jsx)$";
 const BABEL_TRANSFORM_ENTRY_KEY = "^.+\\.(js|jsx|ts|tsx)$";
 
-function overrideBabelTransform(jestConfig, transformKey) {
-    jestConfig.transform[transformKey] = require.resolve("./jest-babel-transform.js");
+function overrideBabelTransform(jestConfig, cracoConfig, transformKey) {
+    // The cracoConfig needs to be available within the jest-babel-transform in order to honor its settings.
+    // This approach is based on https://github.com/facebook/jest/issues/1468#issuecomment-384825178
+    jestConfig.globals = jestConfig.globals || {};
+    jestConfig.globals._cracoConfig = cracoConfig;
```

Comment:
Does the craco config you provide here is still available in the transform script?

If it is, you should be able to remove the code which load the craco config in the transform script?

Reply:
It is, yes.

I initially left the code which loads the craco config intact in case somebody is requiring/using the `jest-babel-transform.js` file directly (without using either a craco.config.js or the API) but in hindsight I doubt anybody is doing that: you'd have to install CRACO but then not actually use it, for that case to ever happen. I'll remove it.

Reply:
Done: that code is now removed